src/VsixV3/Microsoft.Net.Sdk.swr

```diff
@@ -0,0 +1,32 @@
+use vs
+
+package name=Microsoft.Net.Sdk
+        version=$(Version)
+
+vs.localizedResources
+  vs.localizedResource language=en-us
+                       title="Microsoft Net Sdk"
+                       description="SDK for .NET projects"
+
+folder "InstallDir:MSBuild\Sdks\Microsoft.Net.Sdk\sdk"
```

Comment:
I disagree with this, elaborated on your MSBuild PR.

It doesn't matter much here since this applies only to Windows, but it's probably too late to put this behavior in for the MSBuild-to-CLI drop for the next prerelease, and they should be consistent.